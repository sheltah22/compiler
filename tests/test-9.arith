let x:int = 5 in
  if ((2 * x) = 10)
  then x
  else x * x
