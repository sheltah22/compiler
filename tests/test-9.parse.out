(let x : int = 5 in (if (= (* 2 x) 10) x (* x x)))
